src/index.js

```diff
@@ -6,31 +6,44 @@ import * as flow from './utils/flow';
 import * as timers from './utils/timers';
 import { GLOW_VERSION } from './constants';
 import onExit from 'signal-exit';
+import chalk from 'chalk';
+import { Lang } from './Lang';
 
 export default async function glow(opts: GlowOptions) {
-  let cwd = opts.cwd;
-  let start = timers.now();
+  const cwd = opts.cwd;
+  const start = timers.now();
 
   onExit(() => {
-    let end = timers.now();
-    let seconds = timers.seconds(start, end);
+    const end = timers.now();
+    const seconds = timers.seconds(start, end);
     env.logger.info(env.lang.get('doneIn', seconds));
   });
 
-  let flowConfigPath = await flow.getFlowConfigPath(cwd);
-  let flowRootDir = flow.getFlowRootDir(flowConfigPath);
-  let flowBinPath = flow.getFlowBinPath(flowRootDir);
-
-  let env = new Env({
+  const flowConfigPath = await flow.getFlowConfigPath(cwd);
+  const flowRootDir = flow.getFlowRootDir(flowConfigPath);
+  const flowConfig = await flow.getFlowConfig(flowRootDir);
+
+  if (!flowConfig) {
+    console.error(
+      chalk.red.bold(
+        Lang.get(
+          'noFlowBinary',
+          flow.getPossibleFlowBinPaths(flowRootDir).join(', ')
+        )
+      )
+    );
+    process.exit(1);
+    return { errors: [1] };
```

Comment:
This will never be reached, just `throw` if it gives you type errors

Reply:
Doesn't work unfortunately, some flow errors due to https://github.com/Bhullnatik/glow/blob/636d5f14e7eb832392759c470fccf7ba91c1d040/src/cli.js#L91